spec/models/game_spec.rb

```diff
@@ -2,10 +2,16 @@
 
 RSpec.describe Game, type: :model do
 
+  subject { create :game }
+
   context "validations" do
 
-    it { is_expected.to have_many :users }
+    it { is_expected.to belong_to :lobby }
 
+    it "starts with 12 cards on the board" do
+      expect(subject.board.count).to eq 12
+      expect(subject.deck.count).to eq 69
```

Comment:
There are 81 cards in total. 12 of them start on the board and the other 69 are in the deck.